Client/LocalClient.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace Dizda\CloudBackupBundle\Client;
+
+use Symfony\Component\Console\Output\ConsoleOutput;
+
+/**
+ * Class LocalClient.
+ *
+ * @author  David Fuertes 
+ */
+class LocalClient implements ClientInterface
+{
+    private $path;
+
+    /**
+     * @param array $params user
+     */
+    public function __construct($params)
+    {
+        $params     = $params['local'];
+        $this->path = $params['path'];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function upload($archive)
+    {
+        $fileName = explode('/', $archive);
+        $status=copy ($archive, $this->path."/".end($fileName));
```

Comment:
can you leave spaces before and after '=', and remove the space just after copy?
Also using only simple quotes ' for strings